ext/soap/soap.c

```diff
@@ -1716,12 +1726,13 @@ PHP_METHOD(SoapServer, fault)
 	size_t code_len, string_len, actor_len = 0;
 	zval* details = NULL;
 	zend_string *name = NULL;
+	zend_string *lang = ZSTR_EMPTY_ALLOC();
 	soapServicePtr service;
 	xmlCharEncodingHandlerPtr old_encoding;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "ss|szS",
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "ss|szSS",
```

Comment:
You use the PATH specifier for the constructor but only string here for the new lang param